dex-it/src/test/scala/com/wavesplatform/it/sync/OrderFeeTestSuite.scala

```diff
@@ -27,6 +27,14 @@ class OrderFeeTestSuite extends MatcherSuiteBase {
          |    dynamic {
          |      base-fee = $baseFee
          |    }
+         |    percent {
+         |      asset-type = amount
+         |      min-fee = 10
+         |    }
+         |    fixed {
```

Comment:
there need case in fixed mode paying fee in asset which doesn't take part in trading asset pair